onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -145,45 +145,40 @@ class AttentionCPUBase : public AttentionBase {
       const int loop_len = batch_size * num_heads_;
       const float alpha = scale_ == 0.0f ? 1.0f / sqrt(static_cast<float>(head_size)) : scale_;
 
-      // The cost of Gemm
-      const double cost = static_cast<double>(head_size) * sequence_length * total_sequence_length;
```

Comment:
This cost -
mask broadcast (if there is mask): sequence_length * total_sequence_length
for concate past_k : total_sequence_length * v_head_size
matmul: sequence_length * total_sequence_length * head_size
position bias: sequence_length * total_sequence_length

The total cost is about (if we omit the optional cost for mask and bias):
total_sequence_length * v_head_size + sequence_length * total_sequence_length * head_size